diff --git a/src/server.c b/src/server.c
index d6e02b2..ce08064 100644
--- a/src/server.c
+++ b/src/server.c
@@ -4471,6 +4471,9 @@ int processCommand(client *c) {
         }
 
         c->cmd = c->lastcmd = c->realcmd = cmd;
+#ifdef PREFETCHIT_REDIS_CMD_PROC
+        if (c->cmd) __builtin_prefetch((const void *)c->cmd->proc, 0, 3);
+#endif
         sds err;
         if (!commandCheckExistence(c, &err)) {
             rejectCommandSds(c, err);
